### PR TITLE
Fix/teeolu/change icons

### PR DESCRIPTION
**Pull request by teeolu**
This PR changes the Icons on the application from Ionicons to Material-ui icons

### CODE REVIEW

**Review by gbenga504 — DISMISSED** [review]
LGTM

File: src/components/Comments/Comment.tsx [hunk 1]
```diff
@@ -82,7 +88,7 @@ const Comment: React.FC<IProps> = ({
     setIsRepliesVisible(prevIsRepliesVisible => !prevIsRepliesVisible);
   }
 
-  function handleShowOptions(event: React.MouseEvent<HTMLButtonElement>) {
+  function handleShowOptions(event: any) {
```
Comment (gbenga504) [inline src/components/Comments/Comment.tsx]: This should be --> `(event: React.MouseEvent<SVGSVGElement>) `

File: src/components/MonacoEditor/SourceCodeHeading.tsx [hunk 2]
```diff
@@ -45,7 +46,7 @@ const SourceCodeHeading: React.FC<IProps> = ({
     // eslint-disable-next-line
   }, [sourceCodeTitle]);
 
-  function handleShowOptions(event: React.MouseEvent<HTMLButtonElement>) {
+  function handleShowOptions(event: any) {
```
Comment (gbenga504) [inline src/components/MonacoEditor/SourceCodeHeading.tsx]: Same as above

**Review by gbenga504 — APPROVED** [review]
I have resolved all this and push, but please check.

File: src/components/MonacoEditor/SourceCodeHeading.tsx (hunk 2 above)
Comment (teeolu) [reply src/components/MonacoEditor/SourceCodeHeading.tsx]: yeah, it ought to be, but typescript was throwing an error that it was a wrong type

**Pull request merged**